### ISSUE TITLE
Changing the ckey of a mob that already has a client disconnects both affected players

### ISSUE THREAD

**Issue by DamianX** [d3athrow/vgstation13#26718]
#### Steps to reproduce
- Be spessman
- VV your mob, change its ckey to some other connected ckey
- Both players get kicked

The player that was originally inside the mob will return to the lobby when they reconnect.
The player that was meant to replace the old player will be able to control the mob when they reconnect.

Started happening recently.

**Comment by ShiftyRail** [comment]
I don't think this is recent at all, VVing the ckey of someone with already a ckey in it ever really worked well. At any rate, this looks like a byond issue.

**Comment by Exxion** [comment]
The first part is expected. The second part sounds like a BYOND issue, though I guess it could theoretically have something to do with `Login()` or whatever

**Comment by d3athrow** [comment]
this behavior should be handled by byond. if you go editing things manually it will disconnect the original client as they no longer have a mob.

**Comment by DamianX** [comment]
That's fine, though we could handle it by ghosting the player instead.
The bug is that even the NEW ckey gets booted from the server. This definitely used to not be the case.

**Comment by d3athrow** [comment]
test it on 512 and see if it behaves the same as it does now.
